nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/serverbeans/ConfigBeansUtilities.java

```diff
@@ -74,6 +75,13 @@ public ConfigBeansUtilities(Applications paramApps, Domain paramDomain) {
         domain = paramDomain;
     }
 
+    /*
```

Comment:
I wrote it this way to match how others are set, and I'm not sure if the default condition is meant to be null - the other defaults mention that they come from a dtd which I have not been able to find.